build/document-extractor.js

```diff
@@ -342,8 +342,17 @@ function _addSingleSpecialSection($) {
     const [proseSections] = _addSectionProse($);
     return proseSections;
   }
-  const query = dataQuery.replace(/^bcd:/, "");
-  const { browsers, data } = packageBCD(query);
+  let query = dataQuery.replace(/^bcd:/, "");
```

Comment:
It looks like this variable is only read once in the `else` branch.

Could you move it there and make it `const`?